docs/Software Architecture Document/Components View/index.md

```diff
@@ -148,6 +148,21 @@ The view is represented by module and subsystem diagrams that show the system's
     - Not work for Android apps.
     - Not work for apps connected via SDL protocol version lower than 4.
 
+#### Plugin Manager
```

Comment:
@LuxoftAKutsan,
Right now you are not only making SAD based on code, but you are also architecturally designing.
And you responsibility to use the correct forms and names.
Code need to be based on architecture!